core/trino-spi/src/test/java/io/trino/spi/TestPage.java

```diff
@@ -32,6 +33,23 @@
 
 public class TestPage
 {
+    @Test
+    public void testGetSizeInBytes()
+    {
+        BlockBuilder blockBuilder = BIGINT.createBlockBuilder(null, 1);
+        blockBuilder.writeLong(1);
+        Block block = blockBuilder.build();
+        Block lazyBlock = lazyWrapper(block);
+
+        Page lazyPage = new Page(1, lazyWrapper(block), block);
```

Comment:
what's the purpose of loading `lazyBlock` on 46 since `lazyPage` has a different object `lazyWrapper(block)`?  did you mean `lazyBlock` here?